src/checkpoint/checkpoint.h

```diff
@@ -65,43 +65,44 @@ struct __wt_ckpt_thread {
     uint64_t usecs;             /* thread timer */
 };
 
+/*
+ * WT_CKPT_TIME_STATS --
```

Comment:
My suggestion is call it `WT_CKPT_TIMER`. While the values are used in statistics, it is not itself specifically a statistics structure. It is a timer structure describing variations on specific events. 

This also feels like something that could be more general than just checkpoint related. Do we have any other similar uses in other areas? It is fine to have it checkpoint related for now though.